e2e/tests/e2e_happy_path/HappyPath.spec.ts

```diff
@@ -118,9 +118,9 @@ suite('Language server validation', async () => {
     });
 
     test('Error highlighting', async () => {
-        await editor.type(javaFileName, 'textForErrorHighlighting', 30);
+        await editor.type(javaFileName, 'error', 30);
         await editor.waitErrorInLine(30);
-        await editor.performKeyCombination(javaFileName, Key.chord(Key.CONTROL, 'z'));
+        await editor.performKeyCombination(javaFileName, Key.chord(Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE));
```

Comment:
Wouldn't Ctrl+Backspace be nicer?

Reply:
I am not sure about the stability of this approach (key combinations)